README.adoc

```diff
@@ -0,0 +1,154 @@
+= Hello World addon
+
+The "hello world" Anaconda addon is a minimalistic example of an addon.
+The code does not distract addon developers from the vital parts of addon design.
+At the same time, it demonstrates Anaconda's API and functionality provided for addons.
+
+The actual functionality provided by the addon is creating a file
+`/root/hello_world.txt`
+with custom contents on the installed system.
+Users can provide text for the file in GUI and Kickstart.
+Besides adding the text, order of the lines can be reversed.
+
+== Changes in the addon
+
+This version of the addon works with Anaconda 32.19 and brings an important change
+where the code no longer runs in a single process. Instead, there are now two parts:
+The GUI or TUI, and worker service. These two processes communicate using D-Bus.
```

Comment:
I recomend a slight rewording of this section, adding "a" before service & mentioning explicitly UI & worker service each run in separate process:

"The GUI or TUI, and **a** worker service. **The UI & the worker service each run in a separate process,** these two processes communicate using D-Bus."